include/sdf/Box.hh

```diff
@@ -76,6 +77,10 @@ namespace sdf
     public: std::optional<gz::math::Inertiald>
             CalculateInertial(double _density);
 
+    /// \brief Get the Axis-aligned box for this Box.
```

Comment:
For this and other similar comments, please state where the center of the Axis-aligned box is located. In most cases, I believe it will be at the center of the SDFormat shape, but I think that is not always the case for meshes

Reply:
Agreed, done in https://github.com/gazebosim/sdformat/pull/1547/commits/529d2e6b1d4c706093d8548339216fac8122174f